HornsAndHooves-moribus.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   # specify any dependencies here; for example:
-  s.add_dependency "rails",      "~> 5.0"
+  s.add_dependency "rails",      "~> 5.2"
```

Comment:
What depends on 5.2 versus 5? Looks like `clear_changes_information` was added in Rails 4.